lib/mailchimp/getNewsletterSubscriptions.js

```diff
@@ -0,0 +1,56 @@
+const fetch = require('isomorphic-unfetch')
+const logger = console
+const {
+  NewsletterSubscription,
+  getMemberApiUrl,
+  supportedInterestIds
+} = require('./utils')
+
+module.exports.getNewsletterSubscriptions = async (userId, email, roles) => {
```

Comment:
same comment as updateNewsletterSubscription: only take userId and context.
Check me from context with `Roles.userIsMeOrInRoles(user, me, ['supporter, admin'])`
load user by userId, if user doesn't exist, throw